wiki/LearnInPublic.md

```diff
@@ -6,53 +6,53 @@
+หัวข้อย่อยใต้กฎข้อนี้คือ: **พยายามทำให้ดีที่สุด แต่อย่าไปกังวลถ้าทำผิดพลาดซ้ำแล้วซ้ำเล่า** ถ้าคุณรู้สึกไม่มั่นใจว่านี่ตนเองรู้จริงหรือเปล่า รู้สึกเหมือนตัวเองเป็น imposter นั่นแหละดีแล้ว แสดงว่าคุณกำลังผลักดันตัวเอง อย่าคิดว่าตัวเองรู้ทุกอย่าง แต่พยายามทำให้เต็มที่ แล้วปล่อยให้อินเทอร์เน็ตคอยแก้ไขเวลาที่คุณทำผิดพลาด (ซึ่งต้องมีบ้างแหละ) แสดงออกไปเลยว่าเรายังเป็นมือใหม่อยู่
 
 **มีคนบอกว่าคุณห่วย?** ดีแล้ว คุณก็เห็นด้วยกับเขา และขอให้เขาอธิบายอย่างละเอียดไปเลยว่าห่วยตรงไหนบ้าง คุณแค่อยากรู้สึกดีหรืออยากจะเก่งจริงๆ กันแน่ล่ะ? คุณไม่จำเป็นต้องไปเถียงหรือเก็บมาเป็นอารมณ์ แค่กลับไปแล้วพิสูจน์ให้เห็นว่าพวกเขาคิดผิดก็พอ แต่แน่นอนว่าถ้าพวกเขาพูดจาไม่ดีหรือใช้คำพูดหยาบคายใส่ ก็บล็อกมันไปเลย
 
-ได้บอกไปรึยังนะว่าการสอนคือวิธีเรียนรู้ที่ดีที่สุด? **ลองพูดไปด้วยตอนเขียนโค้ดดูสิ** มันอาจจะกดดันหน่อย และผมเองก็ไม่ได้ทำบ่อยนัก แต่เวลาที่ผมสัมภาษณ์งาน ครั้งที่ผมทำได้ดีที่สุดคือครั้งที่ผมเผลอพูดเหมือนตอนสอนคนอื่น แทนที่จะพยายามพิสูจน์ตัวเองอย่างเอาเป็นเอาตาย เราเป็นสัตว์สังคม เราดูออกว่าใครมี confidence (ความมั่นใจในตัวเอง) และใครกำลัง desperate (พยายามจนเกินงาม)
+ได้บอกไปรึยังนะว่าการสอนคือวิธีเรียนรู้ที่ดีที่สุด? **ลองพูดไปด้วยตอนเขียนโค้ดดูสิ** มันอาจจะกดดันหน่อย และผมเองก็ไม่ได้ทำบ่อยนัก แต่เวลาที่ผมสัมภาษณ์งาน ครั้งที่ผมทำได้ดีที่สุดคือครั้งที่ผมเผลอพูดเหมือนตอนสอนคนอื่น แทนที่จะพยายามพิสูจน์ตัวเองอย่างเอาเป็นเอาตาย เราเป็นสัตว์สังคม เราดูออกว่าใครมีความมั่นใจ และใครกำลังสิ้นหวัง
 
-ถึงจุดนึงคุณจะเริ่มมีผู้สนับสนุนคอยช่วยเหลือคุณเอง หากคุณเป็นผู้ที่กระหายอยากเรียนรู้ตัวจริง คนเราจะมองออก และคนเหล่านั้นก็จะอยากช่วยเหลือคุณ คุณไม่ต้องไปบอกเขานะ แต่พวกเขากลายเป็น mentor (ผู้ให้คำปรึกษา) ของคุณไปแล้วล่ะ **สิ่งที่สำคัญมากคือ:** รับโอกาสที่พวกเขามอบให้ ([["Pick Up What They Put Down"]]) คิดซะว่าพวกเขากำลังเสนอเควสให้คุณทำ เมื่อไหร่ที่พวกเขาพูดว่า "มีใครอยากช่วยเรื่อง \_\_\_\_ บ้าง?" คุณต้องเป็นเด็กแถวหน้าสุดที่ยกมือรอตั้งแต่พวกเขายังไม่ทันถามจบ พวกเขาเหล่านี้คือ senior engineer บุคลากรที่เป็นที่ต้องการตัวมากที่สุดในวงการเทคโนโลยี พวกเขาจะสละเวลามาคุยกับคุณแบบตัวต่อตัวถ้าคุณช่วยเหลืองานเขา (บอกเลยว่าพวกเขามีงานให้ช่วยตลอดแหละ) ซึ่งคุณไม่มีทางหาซื้ออะไรแบบนี้ได้หรอก พวกเขาจะเต็มใจสอนคุณฟรีๆ คนส่วนใหญ่มองไม่เห็นโอกาสตรงหน้า แต่ไม่ใช่คุณ
+ถึงจุดหนึ่งคุณจะเริ่มมีผู้สนับสนุนคอยช่วยเหลือคุณเอง ถ้าคุณเป็นผู้ที่กระหายอยากเรียนรู้ตัวจริง คนเราจะมองออก และคนเหล่านั้นก็จะอยากช่วยเหลือคุณ คุณไม่ต้องไปบอกเขานะ แต่พวกเขากลายเป็นพี่เลี้ยงของคุณไปแล้วล่ะ **สิ่งที่สำคัญมากคือ:** รับโอกาสที่พวกเขามอบให้ ([["Pick Up What They Put Down"]]) คิดซะว่าพวกเขากำลังเสนอเควสให้คุณทำ เมื่อไหร่ที่พวกเขาพูดว่า "มีใครอยากช่วยเรื่อง \_\_\_\_ บ้าง?" คุณต้องเป็นเด็กแถวหน้าสุดที่ยกมือรอตั้งแต่พวกเขายังไม่ทันถามจบ พวกเขาที่ว่าอาจเป็น senior engineer อาจเป็นบุคลากรที่เป็นที่ต้องการตัวมากที่สุดในวงการเทคโนโลยี พวกเขาจะสละเวลามาคุยกับคุณแบบตัวต่อตัวถ้าคุณช่วยเหลืองานเขา (บอกเลยว่าพวกเขามีงานให้ช่วยตลอดแหละ) ซึ่งคุณไม่มีทางหาซื้ออะไรแบบนี้ได้หรอก พวกเขาจะเต็มใจสอนคุณฟรี ๆ คนส่วนใหญ่มองไม่เห็นโอกาสนี้ตรงหน้า แต่โชคดีที่นั่นไม่ใช่คุณ

```

Comment:
oh, mentor is พี่เลี้ยง, great choice of word krub

Reply:
👍👍